### PR TITLE
test: Retry disk removal in TestStorageMdRaid.testRaid

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

File: test/verify/check-storage-mdraid [hunk 1]
```diff
@@ -203,7 +203,18 @@ class TestStorageMdRaid(storagelib.StorageCase):
 
         # Remove DISK1.  The spare takes over.
         self.raid_remove_disk(disk1)
-        b.wait_not_present(self.card_row("MDRAID device", name=disk1))
+        try:
+            b.wait_not_present(self.card_row("MDRAID device", name=disk1))
+        except Error:
```
Comment (jelly) [inline test/verify/check-storage-mdraid:208]: # test/verify/check-storage-mdraid:208:16: F821 Undefined name `Error`
# Found 1 error.

I guess `Exception`

**Review by jelly — DISMISSED** [review]
Needs a small Python fixup

File: test/verify/check-storage-mdraid (hunk 1 above)
Comment (martinpitt) [reply test/verify/check-storage-mdraid:208]: No, `testlib.Error` please

Comment (mvollmer) [reply test/verify/check-storage-mdraid:208]: Oh.

File: test/verify/check-storage-mdraid [hunk 2]
```diff
@@ -203,7 +203,18 @@ class TestStorageMdRaid(storagelib.StorageCase):
...
+            # Sometimes disk1 is busy. Try again in that case
+            if b.is_visible("#dialog") and "Device or resource busy" in b.text("#dialog"):
+                self.dialog_cancel()
+                self.dialog_wait_close()
```
[the remark this answers is not in the document]
Comment (mvollmer) [reply test/verify/check-storage-mdraid:212]: The nap is done implicitly by the failing `b.wait-not-present`, and I think a retry is not necessary... hopefully...

[the remark this answers is not in the document]
Comment (martinpitt) [reply test/verify/check-storage-mdraid:212]: Ah right!

**Review by martinpitt — APPROVED** [review]
Cheers! 

Gosh, "computers are predictable" they said. "it's hard to create true randomness" they said. Guess what actually happens all the time.. :exploding_head:

**Comment by martinpitt** [comment]
[One failure in testNotRemovingDisks](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20098-20240227-072910-b938261c-arch-storage/log.html#98), but that's a different test than the one changed here. Good enough.

**Pull request merged**